cli/f3.go

```diff
@@ -183,3 +324,111 @@ func prettyPrintManifest(out io.Writer, manifest *manifest.Manifest) error {
 	}
 	return t.ExecuteTemplate(out, "manifest", manifest)
 }
+
+func prettyPrintFinalityCertificate(out io.Writer, cert *certs.FinalityCertificate, entries gpbft.PowerEntries) error {
```

Comment:
This is going to be _very_ verbose as-is.

Reply:
Now it looks something like this (generated from butterflynet):
```
$ lotus f3 certs list 1..2
Instance:     1
Power Table:
  Next:       bafy2bzacead2duwh7rstrxonruk4z3b63mbytknf7rg2gj2jcn2mg5h4znfy6
  Delta:      None
Finalized Chain:
  Length:     100
  Epochs:     1860-1961
  Chain:
    ├──1860 (length: 2): [bafy2bzacebbct7u4mumyx3tgkp3dw7trwjiw54ipwdypcdjhfq3mvfcim62gc, bafy2bzacea7i3zglwimrzmcvm7kb6vhxajnyd2ldr6jh26mzactzgttv7axlw]
    ├──1861 (length: 2): [bafy2bzacedf6lenkzypruo7ihdvqtz6dem7spn3kepmotzcaw4f5v3dtdcdm2, bafy2bzaceb3nbogzotqwzzgtbeh6iagfzgr6iczxpels2nbquhqoo44mrbxno]
    ├──1862 (length: 2): [bafy2bzacearecpydm4uouys2hl4tl6fuo72cjqo6qd4seci7hpwnc5r7cp5vm, bafy2bzacedstu2mbrupyuxok7zndo2t2x3ihphvzz4q6ubofrlwgnkgidp4qw]
    ├──1863 (length: 1): [bafy2bzaceav2osaffsivlimpohzme6m3vv4czyx7skqwwrktztqoamjxokc7a]
    ├──1864 (length: 3): [bafy2bzacecfuwomvw2gpayoqqxxktsc3gy7r2lsoxsuy6ww2mzvey7nkdjpaq, bafy2bzacec7nzoy3jnoacrs54ua4yfrdumnzdp4e3dg2cukcz3i5w6mivoafa, ...]
    ├──1865 (length: 2): [bafy2bzacedvb2my6kiby5qk4gborzrnl4dun7ovfrdjxcyxtziuqn3qoa4cb2, bafy2bzacebgkby553cqtkcximzrnoovgu3tadu5qyhya55fs32jsmdzwrbfzw]
    ├──1866 (length: 3): [bafy2bzacebwnrmqo75jcddiu7nsdhiq2kfxshdv25hzxf4eh3xb2ilopd5oha, bafy2bzaceaqxosy3ipov42gdifmjo3h6yiauenmuqhmljniedvwifye3y4bgs, ...]
    ├──1867 (length: 2): [bafy2bzacecjwm5o4gpdate4my6qh2m2cvrryfhacne7aknb6eedjlkdjjk62q, bafy2bzacecu2qmtabuqqde57wto2ksiooacsc6yo7ncw253ydd6y3iluj255u]
    ├──1868 (length: 3): [bafy2bzacedjojrzgsv774cs2sxwe2r2zojq6zvj5vdj6kjuexq5ueaodcu2j2, bafy2bzaceawpu2rks6nfgpjn7mf2jk4ip45k33akkv4u52al5ejrmj6rxehmc, ...]
    ├──1869 (length: 3): [bafy2bzacebliuzkamrzbwtvzk5gbnd52gaunr356ptiw5fjmtxol45pw5nbyw, bafy2bzacec3rtkrzulxdueqrmz3cfyxanqtrwu25c2aizoctzhxeiwpfbvzh4, ...]
    └──...omitted the remaining 90 tipsets.
Signed by 3 miner(s).
---
Instance:     2
Power Table:
  Next:       bafy2bzacead2duwh7rstrxonruk4z3b63mbytknf7rg2gj2jcn2mg5h4znfy6
  Delta:      None
Finalized Chain:
  Length:     100
  Epochs:     1961-2060
  Chain:
    ├──1961 (length: 2): [bafy2bzacedia6icdcq3c2mfhzbveahql3godbkgdvsqxuym5zt45d5gte6mqo, bafy2bzacea5aznket4av45r4vwqodsz564fplfhzvqbyhtqbql5x5zmqalp34]
    ├──1962 (length: 2): [bafy2bzacebvnlwsybxu4ay7zej3rxpkippz2d5gzymgynzm6sjmq4vyovcfka, bafy2bzacea5e5lfaxgkkk22t3xezfhj2wgjjbgvfvtqwwu542ekf57cm7yara]
    ├──1963 (length: 1): [bafy2bzacebicdxfdpb4sxgndfhhv6n7ze6iucwqqshl6emvhfeysjh5cku4aa]
    ├──1964 (length: 3): [bafy2bzacedba55mpeqkyug344f4v5tgzzltfguo4jguvtvzdt5d6olg76eta2, bafy2bzacedj75vtc7z3frjlcc2bfrdt3gthaqrggdggbxbpy54hp6qkryxo6w, ...]
    ├──1965 (length: 3): [bafy2bzacecseehd5l3zqcejtocwfyxyrmp6pt6uwivmj7vkuelntkok5m6ic4, bafy2bzaceaup6ocmjqwbv2j5vzwciy2h34fl6dhbbs7gcjruav74qh3s4dl54, ...]
    ├──1966 (length: 2): [bafy2bzacedq7xjnyaws7mpsgo4xryvjlaygarn4hljtzpp23f5i4j4fz25l5m, bafy2bzaceae2fcuovd3tofiw4tyvzjsubufivbgl4aymew7rnrkmgq43p4zmu]
    ├──1967 (length: 2): [bafy2bzacedjgkkgtbeiyi2oztv72ogoen3nsr3dsvxh2qo2vtjmk4pja5ytga, bafy2bzaceawclk4x3ywfdp3voifgbv2nohxd3hq5bpyogxazpzu5ijwc5gpp6]
    ├──1968 (length: 3): [bafy2bzacedzauzjjsesidlnczp5b35aupx4w25whonktmxeixs2uldyztx6bc, bafy2bzacea4d2dflo57ja2ukusuzs5ai3gg6t5lr2oz7eympsk4popgqcmgtk, ...]
    ├──1969 (length: 2): [bafy2bzacea6cq2vs5xmvre6tnigrdae5wbi5mkwzy65l2l6zxehukc7vl3ada, bafy2bzacebij2qea6dwytepzukna4gkojrv6cevssndxd522nmim6ifuteg7c]
    ├──1970 (length: 3): [bafy2bzacebooywyxjahbkredc2ibge2yjbotc5zrie5lqzvtm36333wigqtoo, bafy2bzacecf5z7glkp52eu42f5nyatku2u6s25oichvhmk73cblpae43nr7dm, ...]
    └──...omitted the remaining 90 tipsets.
Signed by 3 miner(s).
```